index.html

```diff
@@ -3096,8 +3089,8 @@ <h3 id="docconformance-attr">
           </tr>
           <tr id="att-readonly" tabindex="-1">
             <th>
-              Any element where the <a data-cite="html/input.html#attr-input-readonly">`readonly`</a>
-              attribute is allowed
+              Any element where the `readonly`
+              attribute is allowed: [^face/readonly^],
```

Comment:
this changes the link
was: https://html.spec.whatwg.org/multipage/input.html#attr-input-readonly
now: https://html.spec.whatwg.org/multipage/custom-elements.html#attr-face-readonly

could link to both, but more importantly should be linking to the previous link as that's what this is more applicable here

Reply:
revised the wording of this cell so both could be linked to